### ISSUE TITLE
Fresh install: env.SetFunction undefined

### ISSUE THREAD

**Issue by ghost** [tyler-sommer/squircy2#3]
```
$ go get -u github.com/tyler-sommer/squircy2
# github.com/tyler-sommer/squircy2/squircy
squircy2/squircy/controller.go:34: env.SetFunction undefined (type *stick.Env has no field or method SetFunction)
```

Looks like some developments in stick became out of sync with squircy2?

**Comment by tyler-sommer** [comment]
Ah, the joys of working with unstable projects :)

Thanks, @Term1nal. I've just now pushed a fix for this in b927fb5. Run `go get -u github.com/tyler-sommer/squircy2` and try again. Let me know if you have any problems!

**Comment by ghost** [comment]
Looks like that did the trick :3 Thanks friend. I look forward to playing with this quite a lot.
